experimental/packages/otlp-exporter-base/src/platform/node/util.ts

```diff
@@ -63,8 +62,7 @@ export function sendWithHttp<ExportItem, ServiceRequest>(
       const err = new OTLPExporterError('Request Timeout');
       onError(err);
     } else {
-      // req.abort() was deprecated since v14
```

Comment:
FWIW there was https://github.com/open-telemetry/opentelemetry-js/pull/3681 recently which re-added node 12 compatibility because of complains.

So maybe keep this here for a bit longer